src/Misc/layoutbin/en-US/strings.json

```diff
@@ -168,6 +169,10 @@
   "ProcessExitCode": "Exit code {0} returned from process: file name '{1}', arguments '{2}'.",
   "Prompt0": "Enter {0}",
   "Prompt0Default1": "Enter {0} (press enter for {1})",
+  "QueueConError": "Message Queue Connect Error: {0}. Retrying in {1} sec.",
+  "QueueConnected": "Message Queue connected.",
+  "QueueError": "Message Queue Error: {0}. Retrying in {1} sec.",
```

Comment:
if GetAgentMessageAsync fails we print QueueError, if ConnectAsync or CreateAgentSessionAsync fails, we print QueueConError. I think we can use the same message for both and I'll delete "QueueError".